Apps/gladys/docker-compose.yml

```diff
@@ -66,22 +66,26 @@ x-casaos:
   main: app
   description:
     # Description in English
-    en_us: "Gladys Assistant is a program that runs on any Linux machine: a PC running Ubuntu, a Raspberry Pi, a NAS, a VPS, a server..."
+    en_us: "Gladys Assistant is a modern, privacy-first & open-source home automation software that runs anywhere."
+    fr_fr: "Gladys Assistant est un logiciel de domotique moderne et respectueux de la vie privée."
   tagline:
     # Short description or tagline in English
-    en_us: A privacy-first, open-source home assistant
+    en_us: "A privacy-first, open-source home assistant."
+    fr_fr: "Le logiciel de domotique open-source qui respecte votre vie privée."
   # Developer's name or identifier
-  developer: "GladysAssistant"
+  developer: "Gladys Assistant"
```

Comment:
This needs to stay like the org on github: GladysAssistant

Reply:
Fixed